src/UserSettingsPage.cpp

```diff
@@ -72,6 +72,7 @@ UserSettings::load(std::optional<QString> profile)
       settings.value(QStringLiteral("user/timeline/enlarge_emoji_only_msg"), false).toBool();
     markdown_       = settings.value(QStringLiteral("user/markdown_enabled"), true).toBool();
     invertEnterKey_ = settings.value(QStringLiteral("user/invert_enter_key"), false).toBool();
+    sedEditing_     = settings.value(QStringLiteral("user/sed_editing"), true).toBool();
```

Comment:
How about you make the sed command a /command? It would be one fewer config option, but it would mean you need to type an additional /. I.e. `/s/element/nheko`.

Reply:
I'm not a huge fan of that. I would prefer to have it be a setting that can be on by default (although I can see that having a command might be nice for some people who have it off by default).

Reply:
I don't think this should be on by default, if it doesn't start with a /. Otherwise sending sed commands will not do what you expect and randomly edit messages, unless you use code tags around it.

So I think this is only mergable if one of the following is true:
- It is off by default
- It only accepts `/s/` commands by default and `s/` commands need to possibly be enabled in the settings.

Reply:
I'd be fine with leaving it off by default.